bridges/python/Pipfile

```diff
@@ -8,5 +8,6 @@ requests = "==2.21.0"
 pytube = "==9.5.0"
 tinydb = "==3.9.0"
 beautifulsoup4 = "==4.7.1"
+tmdbsimple = "==2.2.0"
```

Comment:
This PR is quite old, before we merge, we should use the latest versions of the dependencies used in this package.
Currently the latest version of `tmdbsimple` is `2.8.0`.